molecule_vagrant/test/scenarios/molecule/multi-node/molecule.yml

```diff
@@ -1,41 +1,48 @@
 ---
 dependency:
   name: galaxy
+
 driver:
   name: vagrant
   provider:
-    name: virtualbox
+    name: libvirt
+    parallel: false
+
 platforms:
   - name: instance-1
     box: debian/jessie64
-    interfaces:
-      - auto_config: true
-        network_name: private_network
-        type: dhcp
-      - virtualbox__intnet: test_network
-        network_name: private_network
-        ip: 192.168.0.1
+    networks:
+      - identifier: private_network
+        options:
+          auto_config: true
+          type: dhcp
     groups:
       - foo
       - bar
     memory: 1024
     cpus: 1
-    config_options:
-      synced_folder: true
+    provider_options:
+      qemu_use_session: false
```

Comment:
This ``qemu_use_session`` thing is missing on config_options scenario too, leading to test failure. In fact, I think that it possibly needs to be handled in the template. After all, it's a mix of possibly upstream bug and of a change in fedora packaging:
https://fedoraproject.org/wiki/Changes/Vagrant_2.2_with_QEMU_Session
https://github.com/vagrant-libvirt/vagrant-libvirt/issues/958
With this quick and dirty change, it seems to be fine:
```diff
--- a/molecule_vagrant/modules/vagrant.py
+++ b/molecule_vagrant/modules/vagrant.py
@@ -159,6 +159,12 @@ Vagrant.configure('2') do |config|
       {% for option, value in instance.provider_options.items() -%}
       provider.{{ option }} = {{ ruby_format(value) }}
       {% endfor %}
+      # https://fedoraproject.org/wiki/Changes/Vagrant_2.2_with_QEMU_Session
+      # https://bugzilla.redhat.com/show_bug.cgi?id=1697773
+      # https://github.com/vagrant-libvirt/vagrant-libvirt/issues/958
+      {% if provider == "libvirt" and 'qemu_use_session' not in instance.provider_options %}
+      provider.qemu_use_session = false
+      {% endif %}
 
       # Raw provider args
       {% for arg in instance.provider_raw_config_args -%}
```

Reply:
more thoughts about this: I'm wondering if a fix for this issue needs to be pushed in a different PR, and then you'll have to modify your PR. After all, this has nothing to do with your work. What do you think ? If you agree with me, I'll make the PR.